### ISSUE TITLE
Example 6.3.1 algebra error

### ISSUE THREAD

**Issue by grady** [APEXCalculus/APEXCalculusPTX#155]
[Example 6.3.1: Integrating powers of sine and cosine](https://opentext.uleth.ca/apex-video/sec_trigint.html#ex_trigint0)
has an algebraic mistake in the solution. A sign error is introduced in the second line from the end. Can also be verified by graphing this antiderivative and comparing it to the one you obtain with the straightforward substitution u=sin(x).

https://github.com/APEXCalculus/APEXCalculusPTX/blob/a04e5676a76fc6817ae085ebda996e8e4b0d1120/ptx/sec_trigint.ptx#L64

![image](https://user-images.githubusercontent.com/60004/150848541-0cd2e472-c09c-44b4-99a7-7cf437eb242f.png)

**Comment by sean-fitzpatrick** [comment]
I was sure I had fixed this one in source. Seems not!

**Comment by sean-fitzpatrick** [comment]
Also, just noticed that there is a trailing space between an `xref` and a period in the first paragraph, that causes a line break before the period. This comment is just to remind me to fix that once I open a branch for this.

**Comment by APEXCalculus** [comment]
When Grady first pointed out the error in the problem, my first thought was
"why did we make this so hard?" But I was swamped with other things and
assumed I was missing something obvious.

Example 6.3.1 in the HTML doesn't appear in the old printed text; the old
6.3.1 is now 6.3.2 in the HTML. I *think* we added that exercise as an
introduction to the integration technique so we wouldn't just start with a
Key Idea.

In that case, we *should* let u = sin x. The point of the technique is to
get a single sine or cosine in the integrand, and let u = "the other trig
function". It is very straightforward in this case - no algebra is needed.
Once I see this is our goal, I can then see how it plays out in Example
6.3.2, where algebra is needed to get that single trig function.

In short, we can leave the example there, but solve it the easy way. It
demonstrates the point of needing/wanting a single trig function.

Hope that makes sense.
Greg

**Comment by sean-fitzpatrick** [comment]
That's a good idea. Also useful is to understand that although you get two answers that look very different, they are, in fact, equivalent. (Trig identities are no longer the strong point among students that they once* were!)

*Okay, they never were.

**Comment by sean-fitzpatrick** [comment]
Fix proposed in #194

**Comment by sean-fitzpatrick** [comment]
Closing as completed.
